cmd/podman/pod_rm.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	podRmFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Force removal of a running pod by first stopping all containers, then removing all containers in the pod.  The default is false",
+		},
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "Remove all pods",
+		},
+	}
+	podRmDescription = "Remove one or more pods"
+	podRmCommand     = cli.Command{
+		Name: "rm",
+		Usage: fmt.Sprintf(`podman rm will remove one or more pods from the host. The pod name or ID can be used.
+                            A pod with running or attached containrs will not be removed.
+                            If --force, -f is specified, all containers will be stopped, then removed.`),
+		Description:            podRmDescription,
+		Flags:                  podRmFlags,
+		Action:                 podRmCmd,
+		ArgsUsage:              "",
```

Comment:
This should be:

```go
ArgsUsage: "[POD ...]",
```

because you support removing multiple pods by ID when `--all` is not set.